document/alert/alert.html

```diff
@@ -19,18 +19,18 @@
     <div class="grid-container">
       <header class="container-fluid header-container">
         <a class="btn-link" href="/">
-          <h1 class="header-title">Super UI</h1>
+          <h1 class="header-title head-1">Super UI</h1>
```

Comment:
As a code reviewer unable to understand what exactly is head-1 or text-3, If it's related to sizing maybe add sm,lg-etc or if something else add an explanatory class name.